.github/workflows/request-review.yml

```diff
@@ -0,0 +1,40 @@
+name: Request Review from Team
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  assign_team:
+    runs-on: ubuntu-latest
+
+    steps:    
+    - name: Install dependencies
+      run: npm install @octokit/core@5.1.0
+        
+    - name: Assign Team
+      uses: actions/github-script@v7
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+      with:
+        script: |
+            const Octokit = require('@octokit/core').Octokit;
+            const octokit = new Octokit({
+              request: { fetch: fetch,},
+              auth: process.env.GITHUB_TOKEN,
+            })
+            
+            await octokit.request('POST /repos/{orgs}/{repo}/pulls/{pull_number}/requested_reviewers', {
+              orgs: 'json-schema-org',
+              repo: 'website',
```

Comment:
the `org` name and `repo` name should not be hardcoded use `github.repository_owner` and `github.repository.split('/")[-1]`

[source](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)

Reply:
Done : )